components/Approver/ApproverHeader.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
```

Comment:
We have a Header in HackneyDS. Maybe better to use it instead of this custom one